validator/src/main.rs

```diff
@@ -3567,17 +3562,7 @@ fn tip_manager_config_from_matches(
                 Pubkey::new_unique()
             }),
         tip_distribution_account_config: TipDistributionAccountConfig {
-            payer: {
-                let keypair =
-                    keypair_of(matches, "tip_distribution_account_payer").unwrap_or_else(|| {
-                        if !voting_disabled {
-                            panic!("--tip-distribution-account-payer argument required when validator is voting");
-                        }
-                        Keypair::new()
-                    });
-
-                Arc::new(keypair)
-            },
+            node_identity,
```

Comment:
this won't fix our issue, need to use cluster info here + dynamically grab keypair